config/config-logging.yaml

```diff
@@ -35,34 +35,34 @@ data:
     # to actually change the configuration.
 
     # Common configuration for all Knative codebase
-    zap-logger-config: |
-      {
-        "level": "info",
-        "development": false,
-        "outputPaths": ["stdout"],
-        "errorOutputPaths": ["stderr"],
-        "encoding": "json",
-        "encoderConfig": {
-          "timeKey": "ts",
-          "levelKey": "level",
-          "nameKey": "logger",
-          "callerKey": "caller",
-          "messageKey": "msg",
-          "stacktraceKey": "stacktrace",
-          "lineEnding": "",
-          "levelEncoder": "",
-          "timeEncoder": "iso8601",
-          "durationEncoder": "",
-          "callerEncoder": ""
-        }
+  zap-logger-config: |
+    {
+      "level": "debug",
+      "development": false,
+      "outputPaths": ["stdout"],
+      "errorOutputPaths": ["stderr"],
+      "encoding": "json",
+      "encoderConfig": {
+        "timeKey": "ts",
+        "levelKey": "level",
+        "nameKey": "logger",
+        "callerKey": "caller",
+        "messageKey": "msg",
+        "stacktraceKey": "stacktrace",
+        "lineEnding": "",
+        "levelEncoder": "",
+        "timeEncoder": "iso8601",
+        "durationEncoder": "",
+        "callerEncoder": ""
       }
+    }
 
-    # Log level overrides
-    # For all components except the autoscaler and queue proxy,
-    # changes are be picked up immediately.
-    # For autoscaler and queue proxy, changes require recreation of the pods.
-    loglevel.controller: "info"
-    loglevel.autoscaler: "info"
-    loglevel.queueproxy: "info"
-    loglevel.webhook: "info"
-    loglevel.activator: "info"
+  # Log level overrides
+  # For all components except the autoscaler and queue proxy,
+  # changes are be picked up immediately.
+  # For autoscaler and queue proxy, changes require recreation of the pods.
+  loglevel.controller: "debug"
+  loglevel.autoscaler: "debug"
+  loglevel.queueproxy: "debug"
+  loglevel.webhook: "debug"
+  loglevel.activator: "debug"
```

Comment:
Can you elaborate on this change? What useful messages to we get by enabling knative debug by default? How much larger is the log volume?

Seems like since we're moving off of knative build, we shouldn't need more logs

Reply:
Ah, that's a mistake. Good catch.